frontend/packages/kubevirt-plugin/src/components/virtualization-overview/HighPriorityAlerts.tsx

```diff
@@ -50,21 +47,22 @@ const AlertTitle = ({ alert }) => {
 };
 
 const HighPriorityAlerts: React.FC = () => {
-  const filteredAlerts: ConsoleAlert[] = useFilteredAlerts(isKubeVirtHighPriorityAlert);
+  const [filteredAlerts, alertsLoaded] = useFilteredAlerts(isKubeVirtHighPriorityAlert);
 
   return (
     <div className="kv-high-priority-alerts">
-      {filteredAlerts.map((alert) => {
-        const severity = alert?.labels?.severity as AlertSeverity;
-        return (
-          <Alert
-            isInline
-            variant={asAlertVariant(severity)}
-            title={<AlertTitle alert={alert} />}
-            className="kv-high-priority-alerts__alert"
-          />
-        );
-      })}
+      {alertsLoaded &&
+        filteredAlerts?.map((alert) => {
```

Comment:
```suggestion
        filteredAlerts.map((alert) => {
```

Just wondering, can `filteredAlerts` really be `null` or `undefined` when `alertsLoaded` is `true`?

Reply:
I just checked. No, it can't. Out of curiosity, is there any negative consequence to using optional chaining in situations where getting `null` or `undefined` is impossible aside from someone making the false assumption that it could be?

Reply:
It's not really a big deal generated code wise, e.g. expression `filteredAlerts?.map(fn)` basically compiles to

```ts
( filteredAlerts === null || filteredAlerts === undefined ? undefined : filteredAlerts.map(fn) )
```

so it yields some excessive checks.

For me, the bigger deal is the explicit indication that the variable is potentially null or undefined that reduces the strictness of the contract that `filteredAlerts` must be some array.

Reply:
Gotcha. Thanks! =)